configure.ac

```diff
@@ -1660,10 +1635,8 @@ if test -n "$HOSTRUNNER"; then
 fi
 
 # LIBRARY_DEPS, LINK_PYTHON_OBJS and LINK_PYTHON_DEPS variable
-AS_CASE([$ac_sys_system/$ac_sys_emscripten_target],
-  [Emscripten/browser*], [LIBRARY_DEPS='$(PY3LIBRARY) $(WASM_STDLIB) python.html python.worker.js'],
-  [LIBRARY_DEPS='$(PY3LIBRARY) $(EXPORTSYMS)']
-)
+LIBRARY_DEPS='$(PY3LIBRARY) $(EXPORTSYMS)'
+
```

Comment:
The implication here would be that the `python.worker.js` script is no longer required - Can it be deleted?

Reply:
I'm going to update it a tiny bit, add a separate make target, and move it into `Tools/wasm/emscripten/web_example` in a followup. I'd prefer to leave it alone for now to avoid creating git blame churn.